ui/packages/catalog-ui-search/src/main/webapp/component/tabs/search-form/tabs.search-form.js

```diff
@@ -10,15 +10,17 @@
  *
  **/
  /*global require*/
- var Tabs = require('component/tabs/tabs');
- var MySearchFormCollectionView = require('component/search-form/search-form-tab-container.view');
- var MySearchSharingFormCollectionView = require('component/search-form/forms-sharing/search-form-sharing-tab-container.view');
+ const Tabs = require('component/tabs/tabs');
+ const MySearchFormCollectionView = require('component/search-form/search-form-tab-container.view');
+ const MySearchSharingFormCollectionView = require('component/search-form/forms-sharing/search-form-sharing-tab-container.view');
+ const ResultFormCollectionView = require('component/result-form/result-form-tab-container.view');
 
  module.exports = Tabs.extend({
     defaults: {
         tabs: {
             'My Search Forms': MySearchFormCollectionView,
-            'Shared Templates': MySearchSharingFormCollectionView
+            'Shared Templates': MySearchSharingFormCollectionView,
+            'Result Forms' : ResultFormCollectionView
```

Comment:
Based on what @adimka said, do we want _this_ to say `Data View` ? Doesn't seem like the most intuitive verbiage to me, but this is the first thing the user will see when trying to find these forms.

Reply:
I think discussing this with @TeresaHudson will be the best way to clear up terminology and UX issues.